cdap-ui/cdap-config.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const parser = require('./server/config/parser.js');
+let cdapConfig;
+
+async function getCDAPConfig() {
+  if (cdapConfig) {
+    return cdapConfig;
+  }
+
+  try {
+    cdapConfig = await parser.extractConfig('cdap');
+  } catch (e) {
+    return Promise.reject(e);
+  }
+
+  return cdapConfig;
+}
+
+module.exports = {
+    getCDAPConfig
```

Comment:
check all the files to make sure tabbing level is set with 2 spaces

Reply:
Done